### PR TITLE
Added a preference option for character height.

### PR DESCRIPTION
**Pull request by lizardqueenlexi**
## About The Pull Request

Recently, the option to change characters' heights between one of five presets was added, but was made admin-only - partly, at least, because of concerns over balance if you can make your character's sprite smaller. As it's a fun character detail, and I don't think anyone here would be likely to abuse it just to gain a slight combat advantage, I've implemented it as a preference option.

image

If having this as an option causes problems for whatever reason, we can always disable it again, but I think it's worth testing out.

## Why It's Good For The Game

It adds nice variety if all characters aren't visually the exact same height.

## Changelog

:cl:
add: Added a preference option for character height.
/:cl:

### CODE REVIEW

File: orbstation/modules/client/preferences/height_pref.dm [hunk 1]
```diff
@@ -0,0 +1,23 @@
+//character preference, makes character height selectable from five options
+
+/datum/preference/choiced/height
+	category = PREFERENCE_CATEGORY_SECONDARY_FEATURES
+	savefile_identifier = PREFERENCE_CHARACTER
+	savefile_key = "height"
+	can_randomize = FALSE
+	var/height_list = list(
```
Comment (Jacquerel) [inline orbstation/modules/client/preferences/height_pref.dm:8]: Can this be static? It seems like it could/should be.

Comment (Jacquerel) [reply orbstation/modules/client/preferences/height_pref.dm:8]: Also could use a documenting comment.

File: orbstation/modules/client/preferences/height_pref.dm [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+		"Shortest" = HUMAN_HEIGHT_SHORTEST,
+		"Short" = HUMAN_HEIGHT_SHORT,
+		"Medium" = HUMAN_HEIGHT_MEDIUM,
+		"Tall" = HUMAN_HEIGHT_TALL,
+		"Tallest" = HUMAN_HEIGHT_TALLEST,
+	)
+
+/datum/preference/choiced/height/init_possible_values()
+	return list("Shortest", "Short", "Medium", "Tall", "Tallest")
```
Comment (Jacquerel) [inline orbstation/modules/client/preferences/height_pref.dm:17]: I don't know if this exists in byond but if there's a way to do like `return height_list.keys` so it automatically matches the keys of the list above that would be ideal. Not hugely necessary if it's annoying to do though.

File: tgstation.dme [hunk 3]
```diff
@@ -313,6 +313,7 @@
 #include "code\__DEFINES\orb\actions.dm"
 #include "code\__DEFINES\orb\ishelpers.dm"
 #include "code\__DEFINES\orb\jobs.dm"
+#include "code\__DEFINES\orb\prefs.dm"
```
Comment (Jacquerel) [inline tgstation.dme:316]: This file doesn't seem to exist

File: orbstation/modules/client/preferences/height_pref.dm (hunk 2 above)
Comment (lizardqueenlexi) [reply orbstation/modules/client/preferences/height_pref.dm]: For future reference, assoc_to_keys(list) is how you do this. So, it's super easy.

**Review by Jacquerel — APPROVED** [review]

**Pull request merged**